_posts/2015-10-26-Talk-to-the-business.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+name: Talk-to-the-business
+title: Talk to the business
+date: 2015-10-14 09:35:00 +00:00
+author: Ana Nogal
+canonical:
+    name: my personal blog
+    href: http://www.ananogal.com/blog/talk-to-the-business/
+image:
+    src: /assets/img/custom/blog/DDD_SkillsMatter.jpg
+tags:
+- DDD
+- Learning
+- Ubiquitous language
+- Bounded context
+---
+Almost a month ago, I did this amazing course at [SkillsMatter](https://skillsmatter.com/): ["Eric Evans Domain Driven Design Immersion Workshop"](https://skillsmatter.com/courses/202-domain-model). The course was very intensive and provided me with the opportunity to reinforce a lot of concepts I already knew as well as learn a few more.
+
+One of this concepts was [Ubiquitous Language](http://c2.com/cgi/wiki?UbiquitousLanguage). The first time I hear it from our coach [Alberto Brandolini](https://twitter.com/ziobrando), I must confess I understand "Ambiguous Language". The term "Ubiquitous" it's not a simple one, essentially for a non-native English person like myself. While I was listening to the coach explaining all of this concepts around the "Ubiquitous Language", another one did made its way through: ["Bounded Context"](https://en.wikipedia.org/wiki/Domain-driven_design#Bounded_context).
```

Comment:
The first time I *heard* it .... I must confess I *understood* ...
*especially* for a non-native English *speaker* like myself